content/influxdb/cloud/reference/flux/stdlib/influxdb-sample/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Flux InfluxDB sample package
+list_title: InfluxDB sample package
+description: >
+  The Flux InfluxDB sample package provides functions for downloading and outputting InfluxDB data sample.
+  Import the `influxdata/influxdb/sample` package.
```

Comment:
Hi Scott! I was just wondering if the sample data would later require a full procedural doc?

Reply:
@mluu12 @sanderson a procedure would fit nicely in with "Get started with InfluxDB"...maybe "Explore sample data in UI"? 
While we do use sample data in many procedures, right now, if folks search for "sample data" and land on the "Sample data" page, there's no instruction for how to use/explore sample data in the product.